samples/hello-world/build.gradle

```diff
@@ -0,0 +1,48 @@
+configureAndroidApp(project)
+
+apply plugin: 'kotlin-kapt'
+
+android {
+    compileSdkVersion 30
+
+    defaultConfig {
+        applicationId "com.samples.helloworld"
```

Comment:
```suggestion
        applicationId "com.badoo.ribs.samples.helloworld"
```

Reply:
Applied in 62b2ab220124dc01d6089f1834149085ee7d8a46